test/vcf/debugulator_test.cpp

```diff
@@ -314,23 +314,6 @@ namespace ebi
           util::string_split(output.str(), "\t", columns);
           CHECK(columns[8] == "GT:GH");
       }
-      SECTION("Fix FORMAT duplicate field with 1 sample, different values for all")
```

Comment:
This section can be kept, using the macro CHECK_THROWS_AS in the call to the fixer